packages/webdriverio/src/constants.js

```diff
@@ -436,3 +436,8 @@ export const ERROR_REASON = [
     'ConnectionFailed', 'NameNotResolved', 'InternetDisconnected',
     'AddressUnreachable', 'BlockedByClient', 'BlockedByResponse'
 ]
+
+export const TIME_FACTOR = {
```

Comment:
as this is being only used in the test we can also define it there.

Reply:
Done.